rca/project_styleguide/templates/patterns/molecules/relatedcontent/relatedprogrammes.html

```diff
@@ -0,0 +1,50 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="related-content">
+    <div class="section__row grid">
+        <h2 class="related-content__heading heading heading--two">{{title}}</h2>
+    </div>
+
+    {% if related_sections %}
```

Comment:
Is `related_sections` correct, `related_programmes ` is being assigned in the context? https://github.com/torchbox/rca-wagtail-2019/pull/54/files#diff-7e356f728d6d9ada8c4d7be20f54d5fcR346

Reply:
Also, you don't need the `if` statement